servers/src/main/java/tachyon/master/MasterServiceHandler.java

```
@@ -336,4 +336,20 @@ public long worker_register(NetAddress workerNetAddress, List<Long> totalBytesOn
     return mMasterInfo.registerWorker(workerNetAddress, totalBytesOnTiers, usedBytesOnTiers,
         currentBlockIds);
   }
+
+  @Override
+  public boolean user_setPermission(int fileId, String path, int permission,
+      boolean recursive) throws FileDoesNotExistException, InvalidPathException,
+      TachyonException, TException {
+    //TODO MasterInfo will support it
```

maybe throw `UnsupportedOperationException` for now?